rpc/src/services/protocol/mod.rs

```diff
@@ -461,7 +465,18 @@ fn handle_rpc_response(
         ))),
     }
 }
-
+/*
+#[cached( name="CALL_PROTOCOL_RPC_CACHE",
+type = "TimedSizedCache<Vec<u8>, serde_json::value::Value>",
+create = "{TimedSizedCache::with_size_and_lifespan(250,20)}",
+convert = "{bincode::serialize(&(chain_param,&chain_id,&block_hash,&rpc_request)).unwrap()}",
+result = true)]
+ */
+#[cached( name="CALL_PROTOCOL_RPC_CACHE",
+type = "TimedSizedCache<(String,ChainId,BlockHash,Vec<u8>), serde_json::value::Value>",
+create = "{TimedSizedCache::with_size_and_lifespan(10,20)}",
+convert = "{(chain_param.to_owned(),chain_id.clone(),block_hash.clone(),serde_json::to_vec(&rpc_request).unwrap())}",
```

Comment:
@mambisi 
this is still no good, we cannot cache all calls here, 
just those with **http method** `GET`

Reply:
Ok